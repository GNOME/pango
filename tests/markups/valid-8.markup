<span foreground="blue" alpha="65535" size="xx-small" variant="normal" stretch="ultracondensed">Text</span>
<span fgcolor="blue" fgalpha="50%" size="x-small" font_variant="smallcaps" font_stretch="extracondensed">Text</span>
<span background="blue" background_alpha="1" size="small" stretch="condensed">Text</span>
<span bgcolor="blue" bgalpha="33%" size="medium" stretch="semicondensed">Text</span>
<span background="blue" bgalpha="33%" font_size="large" stretch="normal">Text</span>
<span background="#0000ffaa" font_size="x-large" stretch="semiexpanded">Text</span>
<span font_size="xx-large" stretch="expanded">Text</span>
<span font_size="xx-large" stretch="extraexpanded">Text</span>
<span font_size="xx-large" stretch="ultraexpanded">Text</span>
