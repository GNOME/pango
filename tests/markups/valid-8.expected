Text
Text
Text
Text
Text
Text
Text
Text
Text


---

range 0 4
0 4 scale 0.578704
0 4 variant normal
0 4 stretch ultra-condensed
0 4 foreground #00000000ffff
0 4 foreground-alpha 65535
range 4 5
range 5 9
5 9 scale 0.694444
5 9 variant small-caps
5 9 stretch extra-condensed
5 9 foreground #00000000ffff
5 9 foreground-alpha 32767
range 9 10
range 10 14
10 14 scale 0.833333
10 14 stretch condensed
10 14 background #00000000ffff
10 14 background-alpha 1
range 14 15
range 15 19
15 19 scale 1.000000
15 19 stretch semi-condensed
15 19 background #00000000ffff
15 19 background-alpha 21626
range 19 20
range 20 24
20 24 scale 1.200000
20 24 stretch normal
20 24 background #00000000ffff
20 24 background-alpha 21626
range 24 25
range 25 29
25 29 scale 1.440000
25 29 stretch semi-expanded
25 29 background #00000000ffff
25 29 background-alpha 43690
range 29 30
range 30 34
30 34 scale 1.728000
30 34 stretch expanded
range 34 35
range 35 39
35 39 scale 1.728000
35 39 stretch extra-expanded
range 39 40
range 40 44
40 44 scale 1.728000
40 44 stretch ultra-expanded
range 44 2147483647


---

[0:4] (null) Ultra-Condensed 0
[4:5] (null) Ultra-Condensed 0
[5:9] (null) Extra-Condensed Small-Caps 0
[9:10] (null) Extra-Condensed Small-Caps 0
[10:14] (null) Condensed Small-Caps 0
[14:15] (null) Condensed Small-Caps 0
[15:19] (null) Semi-Condensed Small-Caps 0
[19:20] (null) Semi-Condensed Small-Caps 0
[20:24] (null) Small-Caps 0
[24:25] (null) Small-Caps 0
[25:29] (null) Semi-Expanded Small-Caps 0
[29:30] (null) Semi-Expanded Small-Caps 0
[30:34] (null) Expanded Small-Caps 0
[34:35] (null) Expanded Small-Caps 0
[35:39] (null) Extra-Expanded Small-Caps 0
[39:40] (null) Extra-Expanded Small-Caps 0
[40:44] (null) Ultra-Expanded Small-Caps 0
[44:2147483647] (null) Ultra-Expanded Small-Caps 0
